src/wp-admin/plugins.php

```diff
@@ -87,10 +87,10 @@
 
 				if ( isset( $_GET['redirect_to'] ) ) {
 					// This will occur when a plugin is activated from within a modal.
-					$url = add_query_arg( 'redirect_to', urldecode( $_GET['redirect_to'] ), $url );
+					$url = add_query_arg( 'redirect_to', $_GET['redirect_to'], $url );
```

Comment:
@peterwilsoncc Security review request. Should this be run through `wp_unslash()`?